pkg/cluster/nodes/create.go

```diff
@@ -109,19 +113,24 @@ func CreateExternalLoadBalancerNode(name, image, clusterLabel, listenAddress str
 }
 
 // CreateWorkerNode creates a worker node
-func CreateWorkerNode(name, image, clusterLabel string, mounts []cri.Mount) (node *Node, err error) {
-	node, err = createNode(name, image, clusterLabel, constants.WorkerNodeRoleValue, mounts)
+func CreateWorkerNode(name, image, clusterName, clusterLabel string, mounts []cri.Mount) (node *Node, err error) {
```

Comment:
this API is definitely going to need rework soon, these functions take too many parameters.

this new parameter is really the network name.

Reply:
thanks.I modified them yesterday and have not submitted them yet. :laughing: